Automattic/woocommerce-payments
# Add fraud protection section in settings page

Fixes #3872 

#### Changes proposed in this Pull Request

This PR adds a fraud protection section in the settings page, currently **behind a feature flag**.

#### Testing instructions

1. This feature is currently behind a feature flag, make sure to turn the `isFraudProtectionSettingsEnabled` flag on in the `client/settings/settings-manager/index.js` file
2. Visit the WCPay settings page, and scroll to the bottom
3. You should see a new section like the screenshot below
4. Turn it on and save the settings
5. Reload the page
6. Check that the setting is still on (saved as WP option)

![Screen Shot 2022-03-21 at 10 56 33](https://user-images.githubusercontent.com/12542046/159238836-8335dfbf-17ac-41da-b6bc-22eb9f2330ba.png)

-------------------


**Post merge**

- [x] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) :'QA Testing Not Applicable - Feature is behind a feature flag
